test/integration/rules/aria-empty-table-header/aria-empty-table-header.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```

Comment:
The document element shouldn't be part of the code. This code is injected into the body so all we really need is everything inside the body tag.